label_studio_ml/examples/tesseract/tesseract.py

```diff
@@ -1,37 +1,82 @@
 from PIL import Image
+import io
 import pytesseract as pt
 from label_studio_ml.model import LabelStudioMLBase
 from label_studio_ml.utils import get_image_local_path
 import logging
+import os
+import json
+import boto3
 
 logger = logging.getLogger(__name__)
 global OCR_config
 OCR_config = "--psm 6"
 
+LABEL_STUDIO_ACCESS_TOKEN = os.environ.get("LABEL_STUDIO_ACCESS_TOKEN")
+LABEL_STUDIO_HOST         = os.environ.get("LABEL_STUDIO_HOST")
+
+AWS_ACCESS_KEY_ID     = os.environ.get("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
+AWS_SESSION_TOKEN     = os.environ.get("AWS_SESSION_TOKEN")
+AWS_ENDPOINT          = os.environ.get("AWS_ENDPOINT")
+
+S3_TARGET = boto3.resource('s3',
+        endpoint_url=AWS_ENDPOINT,
+        aws_access_key_id=AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+        aws_session_token=AWS_SESSION_TOKEN,
+        config=boto3.session.Config(signature_version='s3v4'),
+        verify=False)
 
 class BBOXOCR(LabelStudioMLBase):
-    def __init__(self, **kwargs):
-        super(BBOXOCR, self).__init__(**kwargs)
+
+    @staticmethod
+    def load_image(img_path_url):
+
+        print(img_path_url)
```

Comment:
Are these `print` statements intentionally left in? If so I would recommend using `logger.info` and adding a bit more context to explain what's being logged.